lib/src/device_id.dart

```diff
@@ -17,9 +17,13 @@ abstract class DeviceID {
   /// Changes the device ID with merge
   /// This would merge the current user with the new user given
   /// This is a costly operation and should be used only once!
-  Future<String?> changeDeviceIDWithMerge(String newDeviceID);
+  Future<void> changeDeviceIDWithMerge(String newDeviceID);
```

Comment:
those things look like a minor breaking change because we are making a returner function to be a void function

Please also add a changelog file

Reply:
and plus documentation changes

Reply:
this is actually where we want to move. i hate returning a message

Reply:
old method will still return

Reply:
oh I see